ui/src/flux/components/YieldFuncNode.tsx

```diff
@@ -37,6 +37,15 @@ class YieldFuncNode extends PureComponent<Props, State> {
     this.getData()
   }
 
+  public componentDidUpdate(prevProps) {
+    const {script: prevScript} = prevProps
+    const {script: currentScript} = this.props
+
+    if (prevScript !== currentScript) {
+      this.getData()
+    }
```

Comment:
I think this is cleaner:
```
if(prevProps.script !== this.props.script) {
  this.getData()
}
```
pls to kindly consider

Reply:
will update